cli/package.json

```diff
@@ -161,6 +162,10 @@
       "import": "./svelte/dist/cypress-svelte.esm-bundler.js",
       "require": "./svelte/dist/cypress-svelte.cjs.js",
       "types": "./svelte/dist/index.d.ts"
+    },
+    "./xpath": {
+      "require": "./xpath/scripts",
+      "types": "./xpath/src"
```

Comment:
This change indicated this will ship this first-class with each Cypress release as well.  I didn't realize as we are migrating to the mono-repo -- was this change intentional?

Reply:
@admah This change also ships this plugin with the `cypress` CLI in addition to moving to the mono-repo. Just wanted to confirm we were going to support usage for both:
1. pulling from the `cypress` module
```
require('cypress/xpath')
```
2. pulling from standalone dependency
```
require('cypress-xpath');
```

Reply:
Good point @emilyrohrbough . I will remove it from the CLI syncing

Reply:
Yeah, good catch @emilyrohrbough. That is not what we want currently.